contracts/src/core/EigenDABlobVerifier.sol

```diff
@@ -0,0 +1,296 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDABlobVerifier} from "../interfaces/IEigenDABlobVerifier.sol";
+import {IEigenDAThresholdRegistry} from "../interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "../interfaces/IEigenDABatchMetadataStorage.sol";
+import {IEigenDASignatureVerifier} from "../interfaces/IEigenDASignatureVerifier.sol";
+import {EigenDABlobVerificationUtils} from "../libraries/EigenDABlobVerificationUtils.sol";
+import {OperatorStateRetriever} from "lib/eigenlayer-middleware/src/OperatorStateRetriever.sol";
+import {IRegistryCoordinator} from "lib/eigenlayer-middleware/src/RegistryCoordinator.sol";
+import "../interfaces/IEigenDAStructs.sol";
+
+contract EigenDABlobVerifier is IEigenDABlobVerifier {
+
+    IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistry;
+    IEigenDABatchMetadataStorage public immutable eigenDABatchMetadataStorage;
+    IEigenDASignatureVerifier public immutable eigenDASignatureVerifier;
+
+    OperatorStateRetriever public immutable operatorStateRetriever;
+    IRegistryCoordinator public immutable registryCoordinator;
+
+    constructor(
+        IEigenDAThresholdRegistry _eigenDAThresholdRegistry,
+        IEigenDABatchMetadataStorage _eigenDABatchMetadataStorage,
+        IEigenDASignatureVerifier _eigenDASignatureVerifier,
+        OperatorStateRetriever _operatorStateRetriever,
+        IRegistryCoordinator _registryCoordinator
+    ) {
+        eigenDAThresholdRegistry = _eigenDAThresholdRegistry;
+        eigenDABatchMetadataStorage = _eigenDABatchMetadataStorage;
+        eigenDASignatureVerifier = _eigenDASignatureVerifier;
+
+        operatorStateRetriever = _operatorStateRetriever;
+        registryCoordinator = _registryCoordinator;
+    }
+
+    /**
+     * @notice Verifies a the blob is valid for the required quorums
+     * @param blobHeader The blob header to verify
+     * @param blobVerificationProof The blob verification proof to verify the blob against
+     */
+    function verifyBlobV1(
```

Comment:
do existing v1 integrations need to update and call this new method or can they continue using the old verify method?

Reply:
old library will continue to work